src/Components/SnackBarRemove.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Snackbar from '@material-ui/core/Snackbar';
+import MuiAlert from '@material-ui/lab/Alert';
+import { makeStyles } from '@material-ui/core/styles';
+import RemoveCircleOutlineIcon from '@material-ui/icons/RemoveCircleOutline';
+
+function Alert(props) {
+  return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+
+export default function CustomizedSnackbars() {
+  const [open, setOpen] = React.useState(false);
+
+  const handleClick = () => {
+    setOpen(true);
+  };
+
+  const handleClose = (event, reason) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+
+    setOpen(false);
+  };
+
+  return (
+    <div>
+      <Button onClick={handleClick}>
+        <RemoveCircleOutlineIcon color="secondary" />
+      </Button>
+      <Snackbar open={open} autoHideDuration={6000} onClose={handleClose}>
+        <Alert onClose={handleClose} severity="warning">
+          Removed From Favorites
+        </Alert>
+      </Snackbar>
+    </div>
+  );
+}
```

Comment:
you made two files with different names but inside them put (almost) the same function (by the way, with the exact same name for the function..) -->> you could make just one function (SnackBar) and make it get in the props the string you want to render

Reply:
had some trobles with the snackbars. need to solve it.